scripts/CEE/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -42,6 +42,22 @@ function main() {
   oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-acls.sh \
     --bootstrap-server localhost:9096 --list > "${inspectDir}"/kafka-acls.txt 2>&1
 
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-transactions.sh \
+    --bootstrap-server localhost:9096 list > "${inspectDir}"/kafka-transactions-list.txt 2>&1
+
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-transactions.sh \
+    --bootstrap-server localhost:9096 list > "${inspectDir}"/kafka-transactions-list.txt 2>&1
+
+  oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh -c \
+    "awk 'NR > 1 { print \$1 }' ${inspectDir}/kafka-transactions-list.txt 2>&1 | xargs -L 1 ./bin/kafka-transactions.sh --bootstrap-server localhost:9096 describe --transactional-id \$@ > ${inspectDir}/kafka-transactions-describe.txt 2>&1"
+
+  for b in $(seq 0 "${kafka_num_brokers:-3}"); do
+    oc exec -n "${resource_namespace}" "${kafkaStatefulSet}" -c kafka -- sh /opt/kafka/bin/kafka-transactions.sh \
```

Comment:
Is there any reason to do this sequentially? It would be faster to run each  broker in the background.

Reply:
I guess that could be done, we'd just need to await them before tar-streaming the directory into the logs.